### ISSUE TITLE
Clojure.spec is broken

### ISSUE THREAD

**Issue by viebel** [viebel/cljs-analysis-cache#1]
We need to upgrade Clojure.spec analysis cache files.

It used to be done with Lumo by this [script](https://github.com/viebel/cljs-analysis-cache/blob/master/generate-clojure-spec-cache.cljs). But Lumo is deprecated.

Is there a way in Planck to generate the analysis cache files for clojure.spec.alpha namespaces?

The analysis caches for the macro namespaces are built here https://github.com/planck-repl/planck/blob/5a76879f2d11f8c79323bc5685525d867c46ff32/script/build#L112

```bash
planck-c/build/planck -sk planck-cljs/out/macros-tmp -e"(require 'cljs.analyzer)" -e"(do (set! cljs.analyzer/*cljs-warnings* (assoc cljs.analyzer/*cljs-warnings* :undeclared-var false)) nil)" -e "(require-macros 'planck.repl 'planck.core 'planck.shell 'planck.from.io.aviso.ansi 'clojure.template 'cljs.spec.alpha 'cljs.spec.test.alpha 'cljs.spec.gen.alpha 'cljs.test 'cljs.pprint 'cljs.analyzer.macros 'cljs.compiler.macros 'cljs.env.macros)"
```
Related post: https://blog.fikesfarm.com/posts/2016-02-03-planck-macros-aot.html

Is there a way to run a similar command with planck binary (not as part of the build process)?

If I run  the same command with planck binary then no files is generated. I assume it’s because the planck binary already bundles the macro namespaces.

Also, in the related post, Mike mentions that it’s possible to generate JavaScript for macro namespaces using cljs.js . How would it look like?

**Comment by timothypratley** [comment]
@mfikes any suggestions here?

**Comment by timothypratley** [comment]
@viebel it looks like you figured out a way to do it with shadow-cljs : https://github.com/viebel/cljs-analysis-cache/commit/61a8d74e4c316ed0779f887fe217d196da2d275c <== does this solve this issue or is that a different thing?

**Comment by viebel** [comment]
Fixed by  [61a8d74](https://github.com/viebel/cljs-analysis-cache/commit/61a8d74e4c316ed0779f887fe217d196da2d275c)
